site/content/en/docs/tasks/dev/integrate_a_custom_job.md

```diff
@@ -6,63 +6,99 @@ description: >
   Integrate a custom Job with Kueue.
 ---
 
-Kueue integrates with a couple of Jobs, including Kubernetes batch Job, MPIJob, RayJob or JobSet.  
+Kueue has built-in integrations for several Job types, including
+Kubernetes batch Job, MPIJob, RayJob and JobSet.
 
-There are two options for integrating a Job-like CRD with Kueue:
-- As part of the Kueue repository
+There are two options for adding an additional integration for a Job-like CRD with Kueue:
 - Writing an external controller
+- As part of the Kueue repository
```

Comment:
I guess for now, in-repository integration for public and well-adopted CRDs is preferred.  This allows us to bulk-update integrations in case of changes/new features in the core Kueue. Once the feature set in Kueue is more mature external controllers will be more preferred option. For that reason, please restore the original order of options.